src/Objs/Homepage/HomepageEditingConfig.js

```diff
@@ -48,6 +48,10 @@ Scrivito.provideEditingConfig('Homepage', {
       description: 'Needs to be requested under your GA property settings.' +
         ' See https://analytics.google.com for details.',
     },
+    intercomAppId: {
+      title: 'Intercom App Id',
```

Comment:
`Intercom App ID`.

Reply:
Not exactly. We're using sentence case for the labels. So it's `Intercom app ID`. 🤓